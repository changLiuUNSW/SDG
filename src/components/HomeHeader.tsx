import React from 'react';
import styled from 'styled-components/native';

const Container = styled.View`
  padding-top: 40;
  padding-horizontal: 16;
  margin-bottom: 24;
`;

const Heading = styled.Text`
  color: #ffff;
  font-size: 28;
`;

const HomeHeader = () => (
  <Container>
    <Heading>17 GOALS TO {'\n'}TRANSFER OUR WORLD</Heading>
  </Container>
);

export default HomeHeader;
