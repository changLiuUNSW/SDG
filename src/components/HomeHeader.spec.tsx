import * as React from 'react';
import renderer from 'react-test-renderer';

import HomeHeader from './HomeHeader';

const setup = () => {
  const wrapper = () => <HomeHeader />;

  return {
    wrapper
  };
};

it('should render the component correctly', () => {
  const { wrapper } = setup();
  const component = renderer.create(wrapper());
  expect(component.toJSON()).toMatchSnapshot();
});
