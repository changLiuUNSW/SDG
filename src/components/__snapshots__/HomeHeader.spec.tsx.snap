// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 24,
        "paddingHorizontal": 16,
        "paddingTop": 40,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#ffff",
          "fontSize": 28,
        },
      ]
    }
  >
    17 GOALS TO 
    

    TRANSFER OUR WORLD
  </Text>
</View>
`;
